EXAMPLE/group_vars/test_gce_euw1/cluster_vars.yml

```diff
@@ -41,8 +41,9 @@ cluster_vars:
   image: "projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20191113"
   region: &region "europe-west1"
   dns_zone_internal: "c.{{gcp_credentials_json.project_id}}.internal"
-  dns_zone_external: "{%- if dns_tld_external -%}{{_cloud_type}}-{{_region}}.{{app_class}}.{{buildenv}}.{{dns_tld_external}} {%- endif -%}"
-  dns_server: ""                    # Specify DNS server. nsupdate or route53.  If empty string is specified, no DNS will be added.
+  # dns_zone_external: "{%- if dns_tld_external -%}{{_cloud_type}}-{{_region}}.{{app_class}}.{{buildenv}}.{{dns_tld_external}} {%- endif -%}"
+  dns_zone_external: "clusterverse.vtp.com."
```

Comment:
The external zone should be set in the `dns_tld_external` variable, and the loop in `dns_zone_external` preserved.

If  `dns_tld_external` is empty, `dns_zone_external` will then remain empty, and as defined in `config/tasks/main.yml`, no record will be created as per the defined condition below (e.g `dns_tld_external: "clusterverse.vtp.com.")`.

```
when: (cluster_vars.dns_server is defined and cluster_vars.dns_server != "") and cluster_vars.dns_server=="clouddns" and (cluster_vars.dns_zone_external is defined and cluster_vars.dns_zone_external != "")
```

Reply:
Thanks @antoineserrano. Tested and fixed.